frontend/src/app/docs/_components/Sidebar/Sidebar.tsx

```diff
@@ -4,9 +4,9 @@ import Link from 'next/link';
 import React, { useState } from 'react';
 import styles from './Sidebar.module.css';
 import { usePathname } from 'next/navigation';
-import { config } from '@/config/Constants';
+import { configuration } from '@/config/Constants';
```

Comment:
why the name change?

Reply:
Conflicting name with middleware 'config' export